packages/loader/src/gltf/GLTFResource.ts

```diff
@@ -11,13 +11,12 @@ import {
   Skin,
   Texture2D
 } from "@galacean/engine-core";
+import { PrefabResource } from "../prefab";
 
 /**
  * Product after glTF parser, usually, `defaultSceneRoot` is only needed to use.
  */
-export class GLTFResource extends ReferResource {
-  /** glTF file url. */
-  readonly url: string;
+export class GLTFResource extends PrefabResource {
```

Comment:
I can't see many relations between PrefabResource and GLTFResource except a small amount of properties.

Reply:
reverted